service/history/configs/config.go

```diff
@@ -437,6 +442,8 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int32, isAdvancedVis
 		HistoryCountLimitError:             dc.GetIntPropertyFilteredByNamespace(dynamicconfig.HistoryCountLimitError, 50*1024),
 		HistoryCountLimitWarn:              dc.GetIntPropertyFilteredByNamespace(dynamicconfig.HistoryCountLimitWarn, 10*1024),
 
+		EnableCloseWorkflowCleanup: dc.GetBoolPropertyFnWithNamespaceIDFilter(dynamicconfig.EnableCloseWorkflowCleanup, false),
```

Comment:
We are ok with filter by namespaceID instead of name? Seems like a burden to operators.

Reply:
In visibilty task executor, I can get the namespace id from the task, but not the name. I'd have to query the namespace registry.

Reply:
Changed to filter by namespace name.